resallocwebui/templates/pools.html

```diff
@@ -26,7 +26,14 @@ <h1 class="resource--title">Resalloc pools</h1>
     {% for pool, info in information.items()|sort(attribute='0') %}
     <tr>
         <th scope="row">{{ pool }}</th>
-        <td class="col-sm-4">{{ info['DESCRIPTION'] |default(omit, True) }}</td>
+        <td class="col-sm-4">
+            {% if info['DESCRIPTION'] %}
+            <details>
+                <summary>Description</summary>
```

Comment:
Can you put in the summary name of the pool? And then we can have one column instead of two.